easybuild/easyconfigs/s/Spyder/Spyder-5.0.5-foss-2020b.eb

```diff
@@ -0,0 +1,278 @@
+    }),
+    ('binaryornot', '0.4.4', {
+        'checksums': ['359501dfc9d40632edc9fac890e19542db1a287bbcfa58175b66658392018061'],
+    }),
+    ('bleach', '3.3.1', {
+        'checksums': ['306483a5a9795474160ad57fce3ddd1b50551e981eed8e15a582d34cef28aafa'],
+    }),
+    ('cloudpickle', '1.6.0', {
+        'checksums': ['9bc994f9e9447593bd0a45371f0e7ac7333710fcf64a4eb9834bf149f4ef2f32'],
+    }),
+    ('cookiecutter', '1.7.3', {
+        'checksums': ['6b9a4d72882e243be077a7397d0f1f76fe66cf3df91f3115dbb5330e214fa457'],
+    }),
+    ('decorator', '5.0.9', {
+        'checksums': ['72ecfba4320a893c53f9706bebb2d55c270c1e51a28789361aa93e4a21319ed5'],
+    }),
+    ('diff-match-patch', '20200713', {
+        'checksums': ['da6f5a01aa586df23dfc89f3827e1cafbb5420be9d87769eeb079ddfd9477a18'],
+    }),
+    ('flake8', '3.9.2', {
+        'checksums': ['07528381786f2a6237b061f6e96610a4167b226cb926e2aa2b6b1d78057c576b'],
+    }),
+    ('helpdev', '0.7.1', {
+        'checksums': ['bb62a79acbac141dadf42cadeb92bb7450dd18b9824a62043b6a0b149190db3d'],
+    }),
+    ('isort', '5.9.2', {
+        'use_pip': False,
+        'checksums': ['f65ce5bd4cbc6abdfbe29afc2f0245538ab358c14590912df638033f157d555e'],
+    }),
+    ('jedi', '0.17.2', {
+        'checksums': ['86ed7d9b750603e4ba582ea8edc678657fb4007894a12bcf6f4bb97892f31d20'],
+    }),
+    ('jeepney', '0.7.0', {
+        'use_pip': False,
+        'checksums': ['1237cd64c8f7ac3aa4b3f332c4d0fb4a8216f39eaa662ec904302d4d77de5a54'],
+    }),
+    ('arrow', '1.1.1', {
+        'checksums': ['dee7602f6c60e3ec510095b5e301441bc56288cb8f51def14dcb3079f623823a'],
+    }),
+    ('jinja2-time', '0.2.0', {
+        'checksums': ['d14eaa4d315e7688daa4969f616f226614350c48730bfa1692d2caebd8c90d40'],
+    }),
+    ('jupyterlab_pygments', '0.1.2', {
+        'checksums': ['cfcda0873626150932f438eccf0f8bf22bfa92345b814890ab360d666b254146'],
+    }),
+    ('importlib_metadata', '4.6.1', {
+        'checksums': ['079ada16b7fc30dfbb5d13399a5113110dab1aa7c2bc62f66af75f0b717c8cac'],
+    }),
+    ('keyring', '21.8.0', {
+        'preinstallopts': """sed -i 's/setup()/setup(version="%(version)s")/g' setup.py && """,
+        'checksums': ['1746d3ac913d449a090caf11e9e4af00e26c3f7f7e81027872192b2398b98675'],
+    }),
+    ('keyrings.alt', '4.1.0', {
+        'preinstallopts': """sed -i 's/setup()/setup(version="%(version)s")/g' setup.py && """,
+        'checksums': ['52ccb61d6f16c10f32f30d38cceef7811ed48e086d73e3bae86f0854352c4ab2'],
+    }),
+    ('lazy-object-proxy', '1.6.0', {
+        'checksums': ['489000d368377571c6f982fba6497f2aa13c6d1facc40660963da62f5c379726'],
+    }),
+    ('mccabe', '0.6.1', {
+        'checksums': ['dd8d182285a0fe56bace7f45b5e7d1a6ebcbf524e8f3bd87eb0f125271b8831f'],
+    }),
+    ('jupyter_client', '6.1.12', {
+        'checksums': ['c4bca1d0846186ca8be97f4d2fa6d2bae889cce4892a167ffa1ba6bd1f73e782'],
+    }),
+    ('nbclient', '0.5.3', {
```

Comment:
Already included in deps (slightly different version, but maybe still compatible)